Documents/Design/ZTag Context Diagram.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-03-04T21:23:30.520Z" agent="5.0 (Windows)" etag="Vo07pENeFnXYtexveT05" version="21.0.2" type="github"><diagram id="C5RBs43oDa-KdzZeNtuy" name="Page-1">7VtZV9s4FP41eZlz6PFCnPAIBChtZoYptEznTbFlW4NtuZKchV8/V4vjJQYSmoWe4aGNdCVruct375VEzz1P51cM5fHvNMBJz7GCec8d9RzHsQbH8CMpC02xrcFQUyJGAkOrCLfkEZcdDbUgAeaNjoLSRJC8SfRplmFfNGiIMTprdgtp0pw1RxFeIdz6KFml3pNAxJo6dAYV/SMmUVzObHsnuiVFZWezEx6jgM5qJPei554zSoUupfNznEjulXy5v17cJ+MH7+rTX/wH+nr2+e6Pb0d6sMtNPllugeFMvHroyeNdep1FmHy9Gods+OMP8vjlyNVDT1FSGH6ZvYpFyUAcAD9NlTIR04hmKLmoqGeMFlmA5TQW1Ko+Y0pzINpA/BcLsTDKgQpBgRSLNDGtsC22+Nt8ryrfZeVDv6yO5vXG0cLUuEBMnEolAYKfIM6JX5IvSVIOvyb7DJs5LZiPn+GZUQfJl5qKGaZfYZpiWCN0YDhBgkybeoiMOkfLfstPbyiB9TmWsb3+iZnIWJ7bt5pDwC4jLMxXleChUFtGRVLqsIFqDN+EaoQ0E6bRtg8tcs3xZ/oNtq0aDaFuKkGrQ4JeAus/C8gUipEs/nOHIuj0lWMGP/cYMTQB0ZmOMG+tb8fnvf7ZR8SCGXAaPr+BbYWUpb3+aP0RDGnC2pQXP61Wa3GcccqUc5FsyQLiIyEbSChNiBbKnUgCgn9xkaIMhn+k6YQoYhaoz0LpghyLilixI0/QAjP+AYp3MVZeSo2BMzktU/5ITaoaI5Ti9XfTsqVZTAS+zZHSwxk446YdIJ5r7xiSubSnsxCU/ZwmML383A36eBgcK0tg9AHXWobOxPW8jW1iipnA82e1uAwSjptA5XimPqv8rT00tLjua4fWz2u+nfBwEE/xzP2TienkW/z9cby0671C1ZYxpwyWapjTudetu6OfwpxSe9bBnHOagZGiDAlpRK9GgSYE3S64wPsCoIvbG1drOig6oCcJSU/Ghh5Kpf3q/4EypgwtYSOQcPKbXKoPZg6xMUTXwAYJJoym8GPYU6Ib10O+do1qJpwFXAfZIYlUIQUEVDQFmKtTbjD6OQxWZApwecWMCfIfsIJVrkUCWkHkLr9c3N5JjJ1qj0Muyd5Ac1cQ2I7V9guBnYboPW+INS56PwqZwqhQ64grdDuFDradz7UkTHsphTH9ghpSJzQr9YhridfUS3tWhn2Cp6WTlG51qfUvKtpPLPVVZrOl+d7OMC8YEei4aNpNM4bIaIZbAYchcbA8kkVA6Fe1O+UujxzpRbvsU9p9mKj4PSZBgLMV39uO+Xdltcdey2pPOqzW6rBab1dGe/L/TbG2njp1Z9XesCnz4+P9ZtWDdeKjK5VIWDeYkRwSEZSs74+78zG+p3hohKfEV0EAL/xYcxXKKMSPgBhQBqZZKYbkTKmO/6ACkBgJlXwJ2KpKv0wEobOqZg62UM2KzzFSDgV6EYmTdCa9UFhkvnRIKCFC9v3wy8cW7fSqf/jYwn7hZGFHwYWOXgtWJewM5wB4akIkCv4qzX07fvrd3e/U3fetpiF51qHdvb1WtnyqAG2ZqvkxgnGW8Cg1X/7ScEkq9FleoMBYYy5KaBZVyJpTAcwlSK7w9Pxzb5l66gFmL5wCvmuUbnXdVjAx7NAoZ68atXqhUx22KDk51oigiKF0i2IEZxtlMljE0ocDQbIQwDs5NQ0pyEkFq13yfFp2Xlk3i1z3ImxjQdotaHDWdLHuzuTYP0Qm8Np7uLbMGhlEaeq19MHafvpgm3vqF69odnN7B5tVRxplh1zmD/zpNMQedF/uXb6uPxT0Craaq9gHSUffrhLu//rvCWVw+k23Y7dgSFvLSg67sZK25vH2oXSOewile1Z59nYGUoLYvhXIsfZ7CFLusxalQJArL4FakucxymWxSBPdoQotxmiCkxvKicoQ3dGECgEh7GrsIWgrc6eFSEgGYU354KgLPXYWadgtIO8INdy9RoyrJ1LyFkgd3deSiPeQ37T2W/Jz10winV0J0FlNIq/NUVrtNq7rSPFdpqa1lcb11z0Y2J1MnXeAXLTqhwJIZ9VZddhX+WJInr0o7FQDNw6yuTbB8jb83fpMa/v5UFfyvatjuede1Dx/KneG/IcLdQhtXpxscENzS0NxiOcqN5If93jSk68zOJaLR3meVMfsRl/NCw11IxNKtSnfuTHsYzIl6ihxpN+s6IN4DtG1Jhd5b/V5iXkWoMplRPGLX8u0n3wcd2jtzm5lgtT95Mz7Q5YNP3veRTZyxmGH2ho5XtfkqJ/g6Ec3vyYCrYhoXek+KUpv6DURyB3sDIGgWv3FgM6nqj+8cC/+Aw==</diagram></mxfile>
```

Comment:
So - Tapping back in to TTnC Training here, We are directly coupling in this document to "ESP32 with Wifi & Lora " 

Should we replace that with, MCU, with Longer Range Radio, and Shorter range radio comms ? 

![image](https://user-images.githubusercontent.com/11888681/222931119-698fd6ee-e013-40b2-add6-27450d627813.png)

I looked a bit into Lora, And it may not be the *Best* RF for Zombietag. 

![image](https://user-images.githubusercontent.com/11888681/222931113-c7eaec5f-d9ca-4411-aac7-54761da80292.png)

Reply:
Screenshot taken from https://lora.readthedocs.io/en/latest/

Reply:
Apart from that, I agree with the "Flow" of the "States" in this diagram

Reply:
> Should we replace that with, MCU, with Longer Range Radio, and Shorter range radio comms ?

No because thats the HW that i've got and I cba to change it. Maybe if we were making a proper product or somethign that would be worth some more investigation but rn the price & enthusiasm is high enough

Also this project won't beusing the things network so theres no limitation there and i imagine itl;l be within the power limitations of actual lora.